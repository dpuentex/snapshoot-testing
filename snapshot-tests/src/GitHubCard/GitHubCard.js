// src/GitHubCard.js
import React from "react";
import Card from "react-bootstrap/Card";

const GitHubCard = () => {
  return (
    <Card style={{ width: "18rem" }}>
      <Card.Img variant="top" src="your-photo-url" alt="Your Name" />
      <Card.Body>
        <Card.Title>Your GitHub Username</Card.Title>
        <Card.Text>
          A brief blurb about you. This can be anything like your interests,
          what you do, etc.
        </Card.Text>
      </Card.Body>
    </Card>
  );
};

export default GitHubCard;
