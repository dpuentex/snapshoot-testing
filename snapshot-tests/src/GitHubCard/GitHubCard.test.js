// src/GitHubCard.test.js
import React from "react";
import renderer from "react-test-renderer";
import GitHubCard from "./GitHubCard";

test("GitHubCard renders correctly", () => {
  const tree = renderer.create(<GitHubCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
