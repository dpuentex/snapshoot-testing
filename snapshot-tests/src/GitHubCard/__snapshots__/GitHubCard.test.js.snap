// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard renders correctly 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    alt="Your Name"
    className="card-img-top"
    src="your-photo-url"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Your GitHub Username
    </div>
    <p
      className="card-text"
    >
      A brief blurb about you. This can be anything like your interests, what you do, etc.
    </p>
  </div>
</div>
`;
